src/config/config.yml

```diff
@@ -63,19 +63,19 @@ testNumberinitialValue:
   certLetter: A
   sequenceNumber: '000'
 trailerIdInitialValue:
-  trailerId: C530000
+  trailerId: C4000000
   trailerLetter: C
-  sequenceNumber: 530000
+  sequenceNumber: 4000000
 systemNumberInitialValue:
-  systemNumber: '10000000'
+  systemNumber: '4000000'
 plateSerialNumberInitialValue:
-  plateSerialNumber: '0'
+  plateSerialNumber: '4000000'
```

Comment:
We need to check product are okay with this being jumped. Apparently they really care about this value for proof the plate is real etc. Sorry I only found out last week

Reply:
have we seen any conflicts with numbers on plate serial numbers? can this just stay at 0?

Reply:
Checked with Mel, reckons it's best we leave the plates one alone! 😄